src/image.rs

```diff
@@ -430,9 +430,9 @@ pub trait ImageDecoder<'a>: Sized {
     }
 }
 
-/// ImageDecoderExt trait
+/// More exotic image parsing
```

Comment:
This might not be the perfect phrasing, but I didn't come up with anything closer to "The trait that all decoders implement" from `ImageDecoder` that conveys the right thought

Reply:
> Specialized image decoding that need not be supported by all formats.

Reply:
Great! Added.